docs/source/workflow.rst

```diff
@@ -52,77 +52,21 @@ The `Paasta Contract <about/contract.html>`_ describes the
 responsibilities of services that wish to work with PaaSTA.
 
 paasta_tools contains the implementation of several of these rules.
-For example, `generate_deployments_for_service <generate_deployments_for_service.html>`_ is
+For example, `generate_Deployments_for_service <generate_Deployments_for_service.html>`_ is
 the piece that checks each service's git repo for the specially-named branch
 that tells PaaSTA which versions of the service should go to which clusters.
 
 Deployment
 ----------
-A yelpsoa-configs master runs `generate_deployments_for_service <generated/paasta_tools.generate_deployments_for_service.html>`_
-frequently. The generated ``deployments.json`` appears in ``/nail/etc/services/service_name`` throughout the cluster.
-
-Marathon masters run `deploy_marathon_services <deploy_marathon_services.html>`_,
-a thin wrapper around ``setup_marathon_job``.
-These scripts parse ``deployments.json`` and the current cluster state,
-then issue commands to Marathon to put the cluster into the right state
--- cluster X should be running version Y of service Z.
+A yelpsoa-configs master runs `generate_Deployments_for_service <generated/paasta_tools.generate_Deployments_for_service.html>`_
+frequently. The generated ``Deployments.json`` appears in ``/nail/etc/services/service_name`` throughout the cluster.
 
 How PaaSTA Runs Docker Containers
 ---------------------------------
-Marathon launches the Docker containers that comprise a PaaSTA service.
-
-Docker images are run by Mesos's native Docker executor. PaaSTA composes the
-configuration for the running image:
-
-* ``--attach``: stdout and stderr from running images are sent to logs that end
-  up in the Mesos sandbox (currently unavailable).
-
-* ``--cpu-shares``: This is the value set in ``marathon.yaml`` as "cpus".
-
-* ``--memory``: This is the value set in ``marathon.yaml`` as "mem".
-
-* ``--memory-swap``: Total memory limit (memory + swap). We set this to the same value
-  as "mem", rounded up to the nearest MB, to prevent containers being able to swap.
-
-* ``--net``: PaaSTA uses bridge mode to enable random port allocation.
-
-* ``--env``: Any environment variables specified in the ``env`` section will be here. Additional
-  ``PAASTA_``, ``MARATHON_``, and ``MESOS_`` environment variables will also be injected, see the
-  `related docs <yelpsoa_configs.html#env>`_ for more information.
-
-* ``--publish``: Mesos picks a random port on the host that maps to and exposes
-  port 8888 inside the container. This random port is announced to Smartstack
-  so that it can be used for load balancing.
+Kubernetes launches the Docker containers that comprise a PaaSTA service. Once a Pod is scheduled to start, the kubelet on the node running the Pod interacts with the container runtime
```

Comment:
i think that technically they're just containers now since dockershim is out of the way, but that's pretty pedantic (i.e., this is probably fine to leave as-is)